python/doc/source/conf.py

```diff
@@ -77,7 +78,6 @@
 # source_suffix = ['.rst', '.md']
 source_suffix = '.rst'
 
-import glob
```

Comment:
Please double check this one, there might be a reason not to import this earlier.

Reply:
I don't think there is. `glob` is a standard library modules, present on all Python installations.